run.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+
+set_ini_prop() {
+    sed "/\[$2\]/,/^\[/ s/$3\=.*/$3=$4/" -i "${GAMECONFIGDIR}/$1"
+}
+
+set_ini_val() {
+    sed "/\[$2\]/,/^\[/ s/((\"$3\",.*))/((\"$3\", $4))/" -i "/home/steam/$1"
+}
+
+NUMCHECK='^[0-9]+$'
+launchDate=`date +"%Y_%m_%d_%H_%M_%s"`
+
+if [ -f "${GAMECONFIGDIR}/Game.ini" ]; then
+    tar cf "/config/backups/${launchDate}.tar" "/config/saves" "${GAMECONFIGDIR}"
+fi
+
+## Initialise and update files
+if ! [[ "${SKIPUPDATE,,}" == "true" ]]; then
+
+    space=$(stat -f --format="%a*%S" .)
+    space=$((space/1024/1024/1024))
+    printf "Checking available space...%sGB detected\\n" "${space}"
+
+    if [[ "$space" -lt 5 ]]; then
+        printf "You have less than 5GB (%sGB detected) of available space to download the game.\\nIf this is a fresh install, it will probably fail.\\n" "${space}"
+    fi
+
+    printf "Downloading the latest version of the game...\\n"
+
+    /home/steam/steamcmd/steamcmd.sh +force_install_dir /config/gamefiles +login anonymous +app_update "$STEAMAPPID" +quit
+else
+    printf "Skipping update as flag is set\\n"
+fi
+
+mkdir -p "${GAMECONFIGDIR}"
+
+cp /home/steam/*.ini "${GAMECONFIGDIR}"
```

Comment:
Maybe instead of using a Game.ini of this repos making (and subsequent maintenance). Maybe we pull in the base Game.ini file from the devs?  just a curl/wget to use the latest file instead here: https://raw.githubusercontent.com/DreamsideInteractive/FrozenFlameServer/main/Game.ini